django/contrib/admin/static/admin/js/SelectBox.js

```diff
@@ -8,23 +8,56 @@
             const cache = SelectBox.cache[id];
             const boxOptions = box.options;
             const boxOptionsLength = boxOptions.length;
-            for (let i = 0, j = boxOptionsLength; i < j; i++) {
-                const node = boxOptions[i];
-                cache.push({value: node.value, text: node.text, displayed: 1});
+            const noGroup = [];
+            if(box.children.length > 0) {
+                // optgroup not present
+                if(box.children[0].tagName.toLowerCase() === 'option') {
```

Comment:
`.toLowerCase()` is unnecessary computation.

```suggestion
                if(box.children[0].tagName === 'OPTION') {
```

Reply:
removed .toLowerCase()